src/jdk.internal.opt/share/classes/jdk/internal/opt/CommandLine.java

```diff
@@ -46,9 +46,7 @@
  * Various utility methods for processing Java tool command line arguments.
  *
  *  <p><b>This is NOT part of any supported API.
- *  If you write code that depends on this, you do so at your own risk.
- *  This code and its internal interfaces are subject to change or
- *  deletion without notice.</b>
+ *  If you write code that depends on this, you do so at your own risk.</b>
```

Comment:
I think this entire paragraph can be removed. It is "just" the "scary warning" meant to warn people about using not-really-public classes in javac, before we had a module system for the encapsulation.